scripts/install

```diff
@@ -128,7 +128,7 @@ ynh_script_progression --message="Building Node dependencies..." --weight=30
 
 pushd "$final_path"
 	ynh_use_nodejs
-	ynh_exec_warn_less sudo -u $app env $ynh_node_load_PATH GIT_SSH_COMMAND='ssh -o StrictHostKeyChecking=accept-new' npm install
+	ynh_exec_warn_less sudo -u $app env $ynh_node_load_PATH npx yarn 
```

Comment:
the build step automatically runs when starting the service (every time, which could be argued can be improved)

and yarn install is an alias of yarn

i used npx (which will automatically fetch the package and run the command) because i'm not sure if the ynh node install thing has yarn prepackaged

Reply:
Yeah, I know for npx :).

I think the more expressive the commands are, the better. `npx yarn install` is better.

OK for omitting the build step, you're right that it is launch before every run :+1: